l1-contracts/contracts/common/L1ContractErrors.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+error Unauthorized(address caller);
```

Comment:
Better to place errors in the interface version of the contract. So we will see which error supposed to be in which contract + there will be no ridiculously big imports

Reply:
we'll likely end up with the same number of imports unless they live within the interface (but then we need to duplicate them in the tests) and a single file lets us remove duplicates. I don't feel strongly though on this. 

Maybe @dnkolegov does?